Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -180,88 +186,8 @@ - (void)alertView:(UIAlertView *)alertView didDismissWithButtonIndex:(NSInteger)
 {
     if (alertView.firstOtherButtonIndex == buttonIndex) {
         NSString *bugTitle = [alertView textFieldAtIndex:0].text;
-        NSArray *logStores = [self.logStores copy];
```

Comment:
looks like this was all moved down into `_shouldCreateReportWithTitle:`. Were any other changes made to this code?